app/routes.py

```diff
@@ -11,3 +11,30 @@
 location_key = os.environ.get("LOCATION_KEY")
 weather_key = os.environ.get("WEATHER_KEY")
 
+@proxy_bp.route("/location", methods=["GET"])
+def get_lat_lon():
+    loc_query = request.args.get("q")
+    if not loc_query:
+        return {"message": "must provide q parameter (location)"}
+
+    response = requests.get(
+        "https://us1.locationiq.com/v1/search.php",
+        params={"q": loc_query, "key": location_key, "format": "json"}
+    )
+
+    return jsonify(response.json())
```

Comment:
If the response is already in JSON, do we need the `jsonify` call here?

Reply:
I just double checked. This is necessary since LocationIQ returns a list.